lib/repl.js

```diff
@@ -426,7 +497,16 @@ function REPLServer(prompt,
 
   var sawSIGINT = false;
   var sawCtrlD = false;
+  const priorizedSigintQueue = [];
   self.on('SIGINT', function onSigInt() {
+    if (priorizedSigintQueue.length) {
```

Comment:
`if (priorizedSigintQueue.length > 0) {`?  There is no hard rule but core code generally doesn't use implicit type coercion.

Reply:
Done.